README.md

```diff
@@ -1,4 +1,7 @@
 [![codecov](https://codecov.io/gh/valkey-io/valkey/graph/badge.svg?token=KYYSJAYC5F)](https://codecov.io/gh/valkey-io/valkey)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/valkey-io
+/valkey/badge)](https://securityscorecards.dev/viewer/?uri=github.com/valkey-io
+/valkey)
```

Comment:
```suggestion
[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/valkey-io/valkey/badge)](https://securityscorecards.dev/viewer/?uri=github.com/valkey-io/valkey)
```
IS this supposed to be this?

Reply:
I didn't get your question ?

Reply:
I proposed a suggestion, it looks like there is new line which shouldn't be there.

Reply:
Good catch